apps/public-docsite-v9/.storybook/main.utils.js

```diff
@@ -24,7 +24,7 @@ function getVnextStories() {
     .filter(pkgName => pkgName.startsWith('@fluentui/') && !excludedDependencies.includes(pkgName))
     .map(pkgName => {
       const name = pkgName.replace('@fluentui/', '');
-      const storiesGlob = '**/@(index.stories.@(ts|tsx)|*.stories.mdx)';
+      const storiesGlob = '**/@(index.stories.@(ts|tsx)|*.stories.mdx|*.stories.tsx)';
```

Comment:
we moved away from the *.stories.tsx glob to remove all of the invalid files that storybook tries to load. The new pattern is to use /folderName/index.stories.tsx, and then index re-exports all of the foo.stories.tsx files.

Reply:
```suggestion
       const storiesGlob = '**/@(index.stories.@(ts|tsx)|*.stories.mdx)';
```